src/openrct2/rct1/RCT1.h

```diff
@@ -685,8 +686,8 @@ struct rct1_s4
     uint8_t climate;
     uint8_t unk_1F432B;
     uint16_t climate_timer;
-    uint8_t weather;
-    uint8_t target_weather;
+    WeatherType weather;
+    WeatherType target_weather;
```

Comment:
@Gymnasiast  can correct me if I'm wrong, but we don't change `rct1` structs.

Reply:
I will wait for feedback on this before I change it. I would also be interested as into why that would be. Seems like a learning opportunity.

Reply:
No don't change the rct1 structs. Leave as uint8_t. These structures are only used for reading from the binary data. We need to keep the memory layout exactly the same. What is meant to happen is that after reading the data, it should be validated to ensure that it is correct for our type. (we might not do that last step very often). By keeping the types different it ensures that there has to be a conscious decision to validate the type on load.

Reply:
FWIW, I typically define an RCT1/2 counterpart - if it's a `typedef` and not an `enum`, at least.